File: CiscoPCCE.Toolkit/Bean/LicenseReservationConfirmation.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    [XmlType("licenseReservation")]
    public class LicenseReservationConfirmation : BaseApiBean
    {
        [XmlElement("confirmationCode")]
        public string? ConfirmationCode { get; set; }

        [XmlElement("installReservationType")]
        public string? InstallReservationType { get; set; }
    }
}

File: JavaSrc/toolkit/bean/LicenseReservationConfirmation.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


@XmlRootElement(name="licenseReservation")
public class LicenseReservationConfirmation extends BaseApiBean {
  private String confirmationCode;
  private String installReservationType;

  public String getConfirmationCode() {
     return this.confirmationCode;
  }

  public void setConfirmationCode(String confirmationCode) {
     this.confirmationCode = confirmationCode;
  }

  public String getInstallReservationType() {
     return this.installReservationType;
  }

  public void setInstallReservationType(String installReservationType) {
     this.installReservationType = installReservationType;
  }


}
